### DISCUSSION TITLE
Which next OpenSSL release will be long-term?

### DISCUSSION THREAD

**Discussion by xnox** [openssl/openssl#23067] [General Discussion]
Which next OpenSSL release will be designated long-term?

Given predictable release schedule LTS releases should too become predictable.

**Comment by t8m** [comment]
The timing of designating a release to be an LTS one is not yet decided. The current policy is that the next one will be designated after 4 years of designating the previous one at the latest. 3.0 was designated LTS in March 2022. That means under the current rules at the latest in March 2026 we must designate one of the previous releases LTS.

Of course to improve the predictability we might decide to make the rules about LTS more strict, but it did not happen yet. It would be a little bit premature doing that right now as we did not complete any successful time-based release yet.
